### PR TITLE
feat(300): install hab and sd-step in launcher

### PR DESCRIPTION
**Pull request by kumada626**
To use habitat in shared-step, this PR installs habitat in launcher. In addition, it installs sd-step go binary to wrap `hab` commands.
Detail is screwdriver-cd/screwdriver#300 .
This PR will be merged after https://github.com/screwdriver-cd/sd-step is built and released.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -39,6 +39,19 @@ RUN set -x \
...
+   && mkdir /opt/sd/bin \
+   && mv /opt/sd/hab-*-x86_64-linux/hab /opt/sd/bin/hab \
+   && chmod +x /opt/sd/bin/hab \
+   && rm -rf hab-* \
+   # Download sd-step
+   && wget -q -O - https://github.com/screwdriver-cd/sd-step/releases/latest \
+      | egrep -o '/screwdriver-cd/sd-step/releases/download/v[0-9.]*/sd-step' \
+      | wget --base=http://github.com/ -i - -O sd-step\
+   && mv /opt/sd/sd-step /opt/sd/bin/sd-step \
+   && chmod +x /opt/sd/bin/sd-step \
```
Comment (minz1027) [inline Dockerfile]: Do we still need `sd-step` under `/opt/sd`? Why not rm it here?

Comment (kumada626) [inline Dockerfile]: In discussion of screwdriver-cd/screwdriver#300, we planed first pass below.
>First pass:
> - Change launcher's Dockerfile
>    - install hab and it's wrapper command in Dockerfile
>    - install path: /opt/sd/bin/

Therefore, I put sd-step to `/opt/sd/bin`. However, other sd tools under `/opt/sd`. Should we put sd-step to `/opt/sd`?

Comment (minz1027) [inline Dockerfile]: I feel it's more consistent that way :)

Comment (kumada626) [inline Dockerfile]: OK, I'll fix it.

**Pull request merged**